src/main/java/com/bloxbean/cardano/client/address/util/AddressEncoderDecoderUtil.java

```diff
@@ -34,6 +34,8 @@ public static String getPrefixTail(NetworkId network) {
         String prefixTail;
         switch (network) {
             case TESTNET:
+            case PREVIEW:
```

Comment:
We don't need PREVIEW and PREPOD here. Only TESTNET networkId is enough.  So NetworkId enum will have two values TESTNET, MAINNET where TESTNET is for all testnets